mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlConnectorConfig.java

```diff
@@ -45,4 +49,12 @@ public String getConnectionString() {
   public int getDefaultPort() {
     return 3306;
   }
+
+  @Override
+  public Properties getConnectionArgumentsProperties() {
+    Properties prop = super.getConnectionArgumentsProperties();
+    prop.put(JDBC_PROPERTY_CONNECT_TIMEOUT, "20000");
```

Comment:
can you put a comment on what unit this time is?

Reply:
done :)